content/docs/composition-vs-inheritance.md

```diff
@@ -1,22 +1,22 @@
 ---
 id: composition-vs-inheritance
-title: Composition vs Inheritance
+title: Komposisi vs Pewarisan
 permalink: docs/composition-vs-inheritance.html
 redirect_from:
   - "docs/multiple-components.html"
 prev: lifting-state-up.html
 next: thinking-in-react.html
 ---
 
-React has a powerful composition model, and we recommend using composition instead of inheritance to reuse code between components.
+React memiliki model/gaya komposisi yang cukup *powerful*, dan kami merekomendasikan penggunaan komposisi dibandingkan pewarisan (*inheritance*) untuk menggunakan kembali kode antar komponen.
 
-In this section, we will consider a few problems where developers new to React often reach for inheritance, and show how we can solve them with composition.
+Pada bagian ini, kita akan melihat beberapa kasus di mana para pengembang yang baru mengenal React sering kali akan mencoba menerapkan pewarisan, dan menunjukkan bagaimana kita dapat menyelesaikan kasus-kasus tersebut dengan komposisi.
```

Comment:
- _dan_ -> _dan kita akan_ ?
Bagian kalimat terakhir terasa seperti _extension_ dari `kita akan melihat beberapa kasus di mana para pengembang yang baru mengenal React sering kali akan mencoba menerapkan pewarisan`, akhirnya tidak terlihat kalau itu adalah hal berbeda, hal selanjutnya yang "ada di bagian ini". What do you think?